docs/wiki/Contributing.md

```diff
@@ -93,14 +93,6 @@ Throughout the development of Bicep code you should follow the [Bicep Best Pract
 - Each bicep file must contain the below multi-line comment at the very top of the file, with its details filled out:
```

Comment:
Could we re-phrase as this doesn't really make sense after removing the comment? and change ```bicep to where it is relevant?

Reply:
Removed multi-line comment recommendation and adjusted the ```bicep section to targetScope = line. Commit [75bd299](https://github.com/Azure/ALZ-Bicep/pull/218/commits/75bd29944b21aa903768e194389112beb127a83c)